wagtail/admin/templates/wagtailadmin/workflows/confirm_delete.html

```diff
@@ -6,12 +6,12 @@
     {% include "wagtailadmin/shared/header.html" with title=view.page_title subtitle=view.get_page_subtitle icon=view.header_icon %}
 
     <div class="nice-padding">
-        <p>This workflow is currently in progress on {{ workflow_states_in_progress }} pages. Deleting this workflow will cancel moderation on these page.
-            Are you sure you want to delete this workflow?</p>
+        <p>{% trans This workflow is currently in progress on {{ workflow_states_in_progress }} pages. Deleting this workflow will cancel moderation on these page.
```

Comment:
Hey @anujaraj have you tested this locally?
I'm pretty sure the text needs to be in quotation marks - see the docs for reference. Intentionally using old docs as we are still supporting old Django. 

https://docs.djangoproject.com/en/3.0/topics/i18n/translation/#trans-template-tag